code/modules/mob/living/carbon/superior_animal/roach/types/kaiser.dm

```diff
@@ -9,20 +9,22 @@ Has ability of every roach.
 	icon = 'icons/mob/64x64.dmi'
 	icon_state = "kaiser_roach"
 	icon_living = "kaiser_roach"
-	icon_dead = "kiaser_roach_dead"
+	icon_dead = "kaiser_roach_dead"
 	density = TRUE
 
-	turns_per_move = 4
+	turns_per_move = 6
 	maxHealth = 2000
 	health = 2000
 	contaminant_immunity = TRUE
 
 	var/datum/reagents/gas_sac
 
-	melee_damage_lower = 10
-	melee_damage_upper = 20
+	melee_damage_lower = 35
+	melee_damage_upper = 40
```

Comment:
Make it 20 lower and 35 upper limit, don't increase such values too much in one go. It's against contributing guidelines.

Reply:
With this damage it's still pretty weak in terms of its speed/AI and dies to any gun due to not using ranged attacks.
I have written a question about this revision in The Shaft, since I literally changed the values to the amount in the "Orders" section, so this should be better checked with Gray who did the order.

Reply:
Mobs don't suppose to deal too much damage, because they often attack in large numbers. Since Kaiser is supported by other roaches on spawn and by fuhrer-like abbility, it can be too much.

Plus contributing guidelines rule is still a thing.

> Rebalance properly. Don't nerf/buff values of things SIX TIMES in one go. Nerf values for 35-50%, see the results on live server, note them, nerf further when there is still a problem. This is how people in "proper game dev" do that, and we are absolutely not smarter then them. When it's a rework that changes whole area of a game - it's fine to axe values as much as you want, but small changes should not make our codebase look like twitching overdosed hobos. Please do buffs and nerfs by small portions each week.

Reply:
Following the Discord conversation, the min/max damage has been adjusted to 20-35.